src/component/login-form/Login.module.scss

```diff
@@ -0,0 +1,26 @@
+.form{
+    border-radius: 25px;
+  background: #356589;
```

Comment:
@RuthAnn1 , Were you referring to this color?

Reply:
yeah I was just checking to make sure it was the color we wanted.

Reply:
Yeah, please refer to the color pallet found in the `src/assets` folder.
<img width="195" alt="Screenshot 2024-07-26 at 7 25 11 in the evening" src="https://github.com/user-attachments/assets/183c7cdd-6f4d-454a-a345-be70334f3b3d">

Reply:
Noted